src/js/Mixins/Dragging.js

```diff
@@ -364,7 +364,7 @@ const DragMixin = {
       });
 
     if (
-      this._layer instanceof L.Circle ||
+      (this._layer instanceof L.Circle && this._layer.options.editable) ||
```

Comment:
Should it not be `radiusEdit`?

Reply:
Oops, leftover fragment from when this PR first started using `editable` as the option.